src/client.py

```diff
@@ -45,25 +46,28 @@ def _login(email: str, password: str):
                 raise IncorrectCredentials
         return session
 
-    def get_classes(self, target_day: datetime):
+    def get_classes(self, target_day: datetime, family_id: str = None):
         response = self.session.get(
             classes_endpoint(self.box_name),
             params={
                 "box": self.box_id,
                 "day": target_day.strftime("%Y%m%d"),
-                "familyId": "",
+                "familyId": family_id,
+                "_": time.time(),
```

Comment:
what is this for? is strictly necessary?

Reply:
I don't think it is strictly necessary but it's how the actual app make the request so I think we should make it like it.

Reply:
if there's no difference in the response I don't see why having it